gslib/commands/rpo.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module provides the rpo command to gsutil."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import textwrap
+
+from gslib import metrics
+from gslib.command import Command
+from gslib.command_argument import CommandArgument
+from gslib.cs_api_map import ApiSelector
+from gslib.exception import CommandException
+from gslib.exception import NO_URLS_MATCHED_TARGET
+from gslib.help_provider import CreateHelpText
+from gslib.third_party.storage_apitools import storage_v1_messages as apitools_messages
+from gslib.utils.constants import NO_MAX
+
+VALID_RPO_VALUES = ('ASYNC_TURBO', 'DEFAULT')
+VALID_RPO_VALUES_STRING = '({})'.format('|'.join(VALID_RPO_VALUES))
+
+_SET_SYNOPSIS = """
+  gsutil rpo set {} gs://<bucket_name>...
+""".format(VALID_RPO_VALUES_STRING)
+
+_GET_SYNOPSIS = """
+  gsutil rpo get bucket_url...
```

Comment:
This and the _SET_SYNOPSIS appear inconsistent with regards to the bucket name/URL?

Reply:
Done.